### DISCUSSION TITLE
英文缩写 's 间距过大

### DISCUSSION THREAD

**Discussion by WANG-Guangxin** [YunYouJun/valaxy#365] [Q&A]
英文缩写 's 间距过大，应该是默认字体的问题。

除了换字体，有什么别的方案吗？

https://github.com/YunYouJun/hexo-theme-yun/discussions/390

**Comment by leostudiooo** [comment]
弯引号没有专门的 Unicode 码点，这个属于是历史遗留问题，大概只能通过 language 标记和字体选择解决。
或许可以尝试使用 `font-feature-settings: "halt" 1;` 这个 CSS 属性，但同时会压缩所有的中文标点为半宽。如果需要精细控制还是很麻烦。
